content/doc/book/system-administration/FIPS-140.adoc

```diff
@@ -0,0 +1,54 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+ifndef::env-github[:imagesdir: ../../resources/managing]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= FIPS-140
+
+[IMPORTANT]
+----
+support blurb
+----
+
+
+== Jenkins 
+
+It may be possible to run Jenkins in a FIPS-140 compliant nature when the <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, compliance flag>> is enabled, the servlet container is appropriately configured, the JVM is appropriately configured, and the host OS is appropriately configured.
```

Comment:
```suggestion
It may be possible to run Jenkins in a FIPS-140 compliant environment when the <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, compliance flag>> is enabled, and the servlet container, the JVM, and the host OS are all appropriately configured.
```

Reply:
it is not the environment - you can likely run Jenkins in a FIPS compliant environment today - but Jenkins itself will not be FIPS compliant if you do that as it uses non approved encryption libraries.

perhaps `way` is better than `nature`?

```suggestion
It may be possible to run Jenkins in a FIPS-140 compliant way when the <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, compliance flag>> is enabled, and the servlet container, the JVM, and the host OS are all appropriately configured.```

Reply:
```suggestion
It may be possible to run Jenkins in a FIPS-140 compliant manner when the <<managing/system-properties#jenkins-security-FIPS140-COMPLIANCE, compliance flag>> is enabled, and the servlet container, the JVM, and the host OS are all appropriately configured.
```